deployments/linea-goerli/usdc/migrations/1686918613_configurate_and_ens.ts

```diff
@@ -0,0 +1,227 @@
+      {
+        contract: lineaMessageService,
+        signature: 'sendMessage(address,unit256,bytes)',
+        args: [bridgeReceiver.address, 0, l2ProposalData]
+      },
+
+      // 2. Approve Goerli's L1StandardBridge to take Timelock's COMP (for bridging)
+      {
+        contract: COMP,
+        signature: 'approve(address,uint256)',
+        args: [lineaL1TokenBridge.address, COMPAmountToBridge]
+      },
+      // 3. Bridge COMP from Goerli to Linea-Goerli Comet using L1StandardBridge
+      {
+        contract: lineaL1TokenBridge,
+        signature: 'bridgeToken(address,uint256,address)',
+        args: [COMP.address, COMPAmountToBridge, rewards.address]
+      },
+      // 4. Approve Goerli's L1StandardBridge to take Timelock's USDC (for bridging)
+      {
+        contract: USDC,
+        signature: 'approve(address,uint256)',
+        args: [lineaL1TokenBridge.address, USDCAmountToBridge]
+      },
+      // 5. Bridge USDC from Goerli to Linea-Goerli Comet using L1StandardBridge
+      {
+        contract: lineaL1TokenBridge,
+        signature: 'bridgeToken(address,uint256,address)',
+        args: [USDC.address, USDCAmountToBridge, comet.address]
+      },
+      // 6. Update the list of official markets
+      {
+        target: ENSResolverAddress,
+        signature: 'setText(bytes32,string,string)',
+        calldata: ethers.utils.defaultAbiCoder.encode(
+          ['bytes32', 'string', 'string'],
+          [subdomainHash, ENSTextRecordKey, JSON.stringify(officialMarketsJSON)]
+        )
+      }
+    ];
+
+    const description =
+      '# Configurate Linea-Goerli cUSDCv3 market, set reward config, bridge over USDC and COMP, and update ENS text record.';
+    const txn = await govDeploymentManager.retry(async () =>
+      trace(await governor.propose(...(await proposal(goerliActions, description))))
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  async enacted(deploymentManager: DeploymentManager): Promise<boolean> {
+    return true;
+  },
+
+  async verify(
+    deploymentManager: DeploymentManager,
+    govDeploymentManager: DeploymentManager,
+    preMigrationBlockNumber: number
+  ) {
+    const ethers = deploymentManager.hre.ethers;
+    await deploymentManager.spider(); // We spider here to pull in Linea COMP now that reward config has been set
+
+    const { comet, rewards, COMP, USDC } = await deploymentManager.getContracts();
+
+    // 1.
+    const stateChanges = await diffState(comet, getCometConfig, preMigrationBlockNumber);
+    expect(stateChanges).to.deep.equal({
+      baseTrackingSupplySpeed: exp(34.74 / 86400, 15, 18),
+      baseTrackingBorrowSpeed: exp(34.74 / 86400, 15, 18),
+      baseBorrowMin: exp(1, 6),
+      WETH: {
+        supplyCap: exp(1000, 18)
+      }
+    });
```

Comment:
I think these might not be the right values, since you changed other values like `borrowCollateralFactor` as well. Scenarios will likely fail here when verifying the state changes.

Reply:
I changed them back to be the exact same as base goerli.